### PR TITLE
♻️ Traefik rules: final Path needed for viewing files

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- The final missing rule that was missing so that tests for /view entrypoint is again functional

### CODE REVIEW

**Review by GitHK — APPROVED** [review]

File: services/docker-compose.yml [hunk 1]
```diff
@@ -229,7 +229,7 @@ services:
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver.rule=hostregexp(`{host:.+}`)
           && (Path(`/`, `/v0`,`/socket.io/`,`/static-frontend-data.json`,
           `/study/{study_uuid:\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b}`,
-          `/view`) ||  PathPrefix(`/v0/`))
+          `/view`, `/#/view`) ||  PathPrefix(`/v0/`))
```
Comment (pcrespov) [inline services/docker-compose.yml:232]: @sanderegg I am afraid is not the last ... viewers (``studies_dispatcher`` plugin) as a redirection syntax that uses fragments to communicate with the front end.
For instance ``https://osparc.io/#/error?message=Sorry%2C%20I%20could%20not%20find%20this%20&status_code=404``

**Review by pcrespov — CHANGES_REQUESTED** [review]
Let me look at it in more dtail and provide you feedback in amoment

File: services/docker-compose.yml (hunk 1 above)
Comment (pcrespov) [reply services/docker-compose.yml:232]: Actually we use fragments for much more at the root as well
```
https://osparc.io#reset-password?code=131234
https://osparc.io#reset-password?registered
https://osparc.io/#/registration/?invitation={code}
```

**Review by pcrespov — APPROVED** [review]
thx for looking at this L:-)

File: services/docker-compose.yml (hunk 1 above)
Comment (sanderegg) [reply services/docker-compose.yml:232]: ok, so for the /view and /error I will set on top /view/#/ and /error/#/. for the last 3 ones I'm puzzled.

**Pull request merged**